log_backend_net: local_addrX may be used uninitialized

rmelch opened the pull request:
log_backend_net: local_addrX may be used uninitialized

if CONFIG_NET_HOSTNAME_ENABLE=y, local_addrX may be used uninitialized
causing unexpected behaviour.

Signed-off-by: Robert Melchers <robert.melchers@taitradio.com>

rmelch commented:
Could someone fix the commit comment, I can't see what I have done wrong!

`Signed-off-by: Robert Melchers robert.melchers@taitradio.com` looks good to me (now I have added the last name)
Thanks

rlubos commented:
> Could someone fix the commit comment, I can't see what I have done wrong!
> 
> `Signed-off-by: Robert Melchers robert.melchers@taitradio.com` looks good to me (now I have added the last name) Thanks

The commit author name needs to match the signoff entry (you need to use full name in there), currently it's only the first name:
```
Author: Robert <robert.melchers@taitradio.com>
```
Update your `user.name` git configuration, and then update the commit (`git commit --amend --no-edit --reset-author -s`)

rmelch committed:
log_backend_net: local_addrX may be used uninitialized

When CONFIG_NET_HOSTNAME_ENABLE=y, local_addrX may be used uninitialized
causing unexpected behaviour.

Signed-off-by: Robert Melchers <robert.melchers@taitradio.com>
